extras/docker/initDockerLicode.sh

```diff
@@ -115,7 +119,11 @@ run_erizoController() {
 run_erizoAgent() {
   echo "Starting erizoAgent"
   cd $ROOT/erizo_controller/erizoAgent
-  node erizoAgent.js &
+  if [ "$ERIZODEBUG" = "true" ]; then
```

Comment:
`==` or `-eq` ?

Reply:
We're using `=` throughout the whole file but `==` is more POSIX compliant so I'll switch all of them 👍